docker-compose.yml

```diff
@@ -33,29 +65,80 @@ services:
       - "8094"
     container_name: couchbase-sandbox-7.0.0
     profiles:
+      - server
       - local
       - mix-and-match
       - test
 
-  backend:
-    build: .
+  # ---------------------BACKENDS---------------------
+  # cached-backend - Use the `sample-app-python` image
+  # from Docker Hub. The image has preinstalled all of
+  # the dependencies, and contains a copy of the code.
+  #
+  # local-backend - Builds the native Dockerfile, and
+  # mounts the container on this repository.
+  #
+  # cached-capella-backend - Is the same image as the
+  # cached-backend, but uses the Capella user/pass.
+  #
+  # local-capella-backend - Functions the same as the
+  # local-backend, but uses the Capella user/pass.
+
+  cached-backend:
+    image: sample-app-python
```

Comment:
ah, so you don't build the cached-backend.Dockerfile?

Reply:
No, it's not used at all. The only reason it's there is if you wanted to update the image in Docker Hub (there's not really a better place to put it)